playwright-e2e/pages/family-history.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, Page } from '@playwright/test';
+import PageBase from 'pages/page-base';
+import { booleanToYesOrNo } from 'utils/test-utils';
+import { CancerSelector } from './cancer-selector';
+
+
+/**
+ * CMI's family history
+ */
+export class FamilyHistory {
+
+  private readonly page:Page;
+
+/*
+
+await page.getByText('Understanding the history of cancer in someone\'s biological family can help rese').click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.locator('mat-card-content').filter({ hasText: 'Biological / Birth Parent 1: Assigned Female at birth 1 questions answered' }).getByRole('button', { name: 'Edit' }).click();
+  await page.getByTestId('answer:FH_PARENT1_ALIAS').fill('Mom');
+  await page.locator('.picklist-answer-FH_PARENT1_LIVING').getByText('Yes').click();
+  await page.locator('.picklist-answer-FH_PARENT1_AGE_RANGE').getByRole('combobox').selectOption('90-94');
+  await page.locator('.picklist-answer-FH_PARENT1_HAD_CANCER').getByText('Yes').click();
+
+*/
+
+constructor(page: Page) {
+    this.page = page;  
+  }
+
+  async waitForReady(): Promise<void> {
+    await this.page.getByText('Understanding the history of cancer in someone\'s biological family can help rese').click();
+    await this.page.getByText("Tell us about the history of cancer in your biological (blood-related) family").click();
+  }
+
+  async next(): Promise<void> {
+    await this.page.getByRole('button').getByText('Next').click();
+  }
+  
+  async parent(parentNumber:number, p: FamilyMember): Promise<void> {
```

Comment:
I think it's specific to CMI at the moment.  If we see it in another study, I'd be up for trying to generalize it better so that we could embed different questions inside of the modals for different family members.